src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -456,6 +456,11 @@ private void CheckAccessibility(Location location, DiagnosticBag diagnostics, bo
                                                                DeclarationModifiers.AccessibilityMask;
                 }
             }
+            else if (isInterface)
+            {
+                Debug.Assert(explicitInterfaceImplementation);
+                allowedModifiers |= DeclarationModifiers.Abstract;
```

Comment:
This is the `else` part of an `if (!explicitInterfaceImplementation)`, so this is a bit redundant.

Reply:
> This is the else part of an if (!explicitInterfaceImplementation), so this is a bit redundant.

It is here for the benefit of the reader and to prevent unintentional break in the future.

---
In reply to: [285294377](https://github.com/dotnet/roslyn/pull/35756#discussion_r285294377) [](ancestors = 285294377)